gcp/appengine/frontend_handlers.py

```diff
@@ -50,6 +50,15 @@
 _VALID_BLOG_NAME = _WORD_CHARACTERS_OR_DASH
 _VALID_VULN_ID = _WORD_CHARACTERS_OR_DASH
 _BLOG_CONTENTS_DIR = 'blog'
+_DEPS_BASE_URL = 'https://deps.dev'
+_OSV_TO_DEPS_ECOSYSTEMS_MAP = {
```

Comment:
Please add this map to https://github.com/google/osv.dev/blob/master/osv/ecosystems/_ecosystems.py so that all ecosystem-related metadata/configuration is living in the one central location.

Reply:
Updated!